bin/cli/src/main.rs

```diff
@@ -0,0 +1,155 @@
+use clap;
+use clap::{command, Parser};
+use near_da_http_api_data::ConfigureClientRequest;
+use near_da_rpc::near::config::Config;
+use near_da_rpc::near::Client;
+use near_da_rpc::{CryptoHash, DataAvailability};
+use serde::{Deserialize, Serialize};
+use std::fmt::Display as FmtDisplay;
+use std::str;
+use tracing::debug;
+#[derive(Parser, Debug)]
+#[command(author, version, about, long_about = None)]
+struct Args {
+    #[clap(
+        short = 'c',
+        long = "config",
+        help = "Path to the client configuration. If not specified, the client can be configured via PUT /config after starting the server.",
+        default_value = "./bin/cli/da_config.json"
+    )]
+    config: Option<String>,
+    #[command(subcommand)]
+    command: Commands,
+}
+struct AppState {
+    client: Option<Client>,
+}
+
+fn config_request_to_config(request: ConfigureClientRequest) -> Result<Config, anyhow::Error> {
```

Comment:
Ahh! I see why you did this. Since `ConfigureClientRequest` is the only struct in `http-api-data`, i think we can move that into `near-da-primitives` and create a `From<ConfigureClientRequest> for Config` implementation. We can also make a ticket for that since it's just tech debt

Reply:
I'll add some issues to fix all these in another PR